### PR TITLE
Add Pathfinders corp Collegium Corpus

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/pathfinders/CollegiumCopernicus.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      !this.player.getActionsThisGeneration().has(CardName.COLLEGIUM_COPERNICUS);
+  }
+
+  public optionText() {
+    return 'Pay 3 Data (use Collegium Copernicus action)';
+  }
+
+  public trade(colony: Colony) {
+    // TODO(kberg): find a way to remove ! or just suck it up and use an if(!undefined)
+    this.collegiumCopernicus!.resourceCount -= tradeCost(this.player);
```
Comment (bafolts) [inline src/cards/pathfinders/CollegiumCopernicus.ts:107]: I would put this in before merging. With obfuscated javascript this error could show in logs as `unable to read property resourceCount of undefined` and we may not know line number to know which class it came from.

**Review by bafolts — APPROVED** [review]
Discussed offline but I would low the image quality of `corp-logo-colleguim-copernicus.png`. It is currently 168kb which is unnecessary.

File: src/cards/pathfinders/CollegiumCopernicus.ts (hunk 1 above)
Comment (kberg) [reply src/cards/pathfinders/CollegiumCopernicus.ts:107]: Yeah. Done.

**Comment by kberg** [comment]
> Discussed offline but I would lower the image quality of `corp-logo-colleguim-copernicus.png`. It is currently 168kb which is unnecessary.

Before: 172,328
After: 13,193

Thanks for calling me out on it.

**Pull request merged**